### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by saurabhagas**

### CODE REVIEW

File: dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/AttachCommand1x3IT.java [hunk 1]
```diff
@@ -62,6 +62,7 @@ public void test_attach_to_activated_cluster_with_offline_node() throws Exceptio
     assertThat(getUpcomingCluster("localhost", getNodePort(1, 1)).getNodeCount(), is(equalTo(2)));
     assertThat(getUpcomingCluster("localhost", getNodePort(1, 2)).getNodeCount(), is(equalTo(2)));
 
+    waitForPassives(1);
```
Comment (mobasherul) [inline dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/AttachCommand1x3IT.java:65]: waitForNPassives is used in setup right ?

Comment (mobasherul) [reply dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/AttachCommand1x3IT.java:65]: Similar comments in other places

File: dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/AttachInConsistency1x4IT.java [hunk 2]
```diff
@@ -95,6 +95,7 @@ public void testAttachNodeFailAtPrepare() throws Exception {
 
   @Test
   public void attachNodeFailingBecauseOfNodeGoingDownInPreparePhase() throws Exception {
+    waitForNPassives(1, 2);
```
Comment (mobasherul) [inline dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/AttachInConsistency1x4IT.java:98]: Isn't it taken care of in setup method ?

File: dynamic-config/testing/system-tests/src/test/resources/config-property-files/single-stripe_invalid1.properties [hunk 3]
```diff
@@ -14,23 +14,5 @@
 # limitations under the License.
 #
 
-client-lease-duration=50s
-client-reconnect-window=100s
```
Comment (mathieucarbou) [inline dynamic-config/testing/system-tests/src/test/resources/config-property-files/single-stripe_invalid1.properties:18]: Non default value removed

File: dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/AttachCommand1x3IT.java (hunk 1 above)
Comment (saurabhagas) [reply dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/AttachCommand1x3IT.java]: All fixed.

**Comment by saurabhagas** [comment]
> Approving the config files cleanup. With the new handling of the user inputs in the model this gets simplified. Thanks!
> Form the waitForPassive calls, please check: some calls are not necessary.

All fixed!

File: dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/DetachCommand1x2IT.java [hunk 4]
```diff
@@ -42,6 +43,11 @@ public DetachCommand1x2IT() {
     super(Duration.ofSeconds(180));
   }
 
+  @Before
+  public void setUp() {
+    waitForPassives(1);
```
Comment (ramsai1729) [inline dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/DetachCommand1x2IT.java:48]: we already wait for actives and passives when auto activated, why do we need this?

         // wait for server startup if auto-activated
            if (clusterDef.autoStart() && clusterDef.autoActivate()) {
              for (int stripeId = 1; stripeId <= clusterDef.stripes(); stripeId++) {
                waitForActive(stripeId);
                waitForPassives(stripeId);
              }
            }

File: dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/RepairCommand1x2IT.java [hunk 5]
```diff
@@ -229,7 +229,7 @@ public void test_repair_detached_node_restarting_as_active() throws Exception {
     // we can reset this node
     invokeConfigTool("repair", "-f", "reset", "-s", "localhost:" + getNodePort(1, passiveId));
     waitForStopped(1, passiveId);
-    startNode(1, passiveId, "--failover-priority", "availability");
+    startNode(1, passiveId, "--failover-priority", "availability", "-r", tmpDir.getRoot().resolve("config").toString());
```
Comment (ramsai1729) [inline dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/RepairCommand1x2IT.java:232]: what about other defaults? ports etc

File: dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/DetachCommand1x2IT.java (hunk 4 above)
Comment (saurabhagas) [reply dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/DetachCommand1x2IT.java]: Right, missed this part. Removed the first commit as it's not needed as we have the right guards in place.

File: dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/RepairCommand1x2IT.java (hunk 5 above)
Comment (saurabhagas) [reply dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/RepairCommand1x2IT.java]: Fixed.

Comment (ramsai1729) [reply dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/RepairCommand1x2IT.java]: I think  `startNode(1, passiveId);` should work as the config is reset.

Comment (saurabhagas) [reply dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/RepairCommand1x2IT.java]: There's a problem currently with that on Windows.

Comment (ramsai1729) [reply dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/RepairCommand1x2IT.java]: okay, lets add a note to change when the issue is fixed

Comment (saurabhagas) [reply dynamic-config/testing/system-tests/src/test/java/org/terracotta/dynamic_config/system_tests/activated/RepairCommand1x2IT.java]: Done. Running tests locally at the moment. Will push after that

**Pull request merged**